100_Numpy_exercises_with_solutions.md

```diff
@@ -682,7 +682,7 @@ print(F)
 
 w,h = 16,16
 I = np.random.randint(0,2,(h,w,3)).astype(np.ubyte)
-F = I[...,0]*256*256 + I[...,1]*256 +I[...,2]
```

Comment:
The first term `I[...,0]*256*256`  will overflow to zero due to the type of this term will be promoted to `uint16`  not `uint32`.